corehq/apps/es/case_search.py

```diff
@@ -139,7 +139,7 @@ def sort_by_case_property(self, case_property_name, desc=False):
 
 class ElasticCaseSearch(ElasticDocumentAdapter):
 
-    _index_name = getattr(settings, "ES_CASE_SEARCH_INDEX_NAME", "case_search_2018-05-29")
+    _index_name = getattr(settings, "ES_CASE_SEARCH_INDEX_NAME", "case_search_2022-10-14")
```

Comment:
I don't think this is quite right either. I think you want to change the setting in the commcare cloud enviroments file, so it changes only for production. Otherwise it will update the index name for all enviroments that do not specify, which I imagine is almost all of them. That will break all the environments that have not reindexed and therefore do not have this new index name. That is why this default already doesn't match prod, where the index was last reindexed in 2021.

Does that sound right @snopoke

Reply:
ah, good point. You are right @calellowitz

Reply:
ok, I'll revert this and make a cc-cloud PR to update `ES_CASE_SEARCH_INDEX_NAME` in production/public.yml